.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "tools/cocos2d-console"]
         path = tools/cocos2d-console
-        url = git://github.com/cocos2d/cocos2d-console.git
```

Comment:
Should not change it.

Reply:
thanks for mention it, I will revert this file after corresponding https://github.com/cocos2d/cocos2d-console/pull/438 merged.